packages/puppeteer-core/src/bidi/Frame.ts

```diff
@@ -267,11 +283,7 @@ export class BidiFrame extends Frame {
       this.waitForNavigation(options),
       this.browsingContext.navigate(
         url,
-        // Some implementations currently only report errors when the
-        // readiness=interactive. This also ensures that old frames have been
-        // removed.
-        // Related: https://bugzilla.mozilla.org/show_bug.cgi?id=1846601
-        Bidi.BrowsingContext.ReadinessState.Interactive
+        Bidi.BrowsingContext.ReadinessState.None
```

Comment:
This is not only about frames but also about error reporting.

Reply:
I've updated the comment.